cpptests/t_client.cpp

```diff
@@ -29,10 +31,18 @@ class Client {
         }
     }
 
+    Client& operator=(const Client &other_) {
```

Comment:
This will cause a crash, since the internal `ctx` will be freed twice.

Reply:
changed to memcpy